src/Illuminate/Support/Arr.php

```diff
@@ -4,6 +4,7 @@
 
 use ArrayAccess;
 use InvalidArgumentException;
+use Illuminate\Collection\Collection;
```

Comment:
This would mean `illuminate/support` now depends on `illuminate/collection` (which make it a recursive dependency). 

I'm totally against recursive dependency at all cause, one of the easiest way to break composer.

Reply:
So, the best to do is keep things as it is?

Reply:
One of the main reason I separate `orchestra/testbench` to `orchestra/testbench-core` was to allow Laravel Framework to required it and keep `orchestra/testbench` to have `require laravel/framework`. Recursive dependencies is at best unpredictable.